compass/ocean/mesh/lts_regions.py

```diff
@@ -0,0 +1,310 @@
+import math
+import subprocess as sp
+
+import netCDF4 as nc
+import numpy as np
+import xarray
+from shapely.geometry import Point
+from shapely.geometry.polygon import Polygon
+
+from compass.step import Step
+
+
+class LTSRegionsStep(Step):
+    """
+    A step for adding LTS regions to a global MPAS-Ocean mesh
```

Comment:
This class has several files that seem very specific to the `Hurricane` test group so it's not clear to me that it belongs in the general ocean framework.

Reply:
I think we resolved this during our call.